src/Sentry/Scope.cs

```diff
@@ -78,11 +82,76 @@ internal SentryId LastEventId
         /// <see cref="Evaluate"/>
         internal event EventHandler? OnEvaluating;
 
+        /// <inheritdoc />
+        [DataMember(Name = "level", EmitDefaultValue = false)]
+        public SentryLevel? Level { get; set; }
+
+        /// <inheritdoc />
+        [DataMember(Name = "transaction", EmitDefaultValue = false)]
+        public string? Transaction { get; set; }
+
+        [DataMember(Name = "request", EmitDefaultValue = false)]
+        private Request? _request;
+
+        /// <inheritdoc />
+        public Request Request
+        {
+            get => _request ??= new Request();
+            set => _request = value;
+        }
+
+        [DataMember(Name = "contexts", EmitDefaultValue = false)]
+        private Contexts? _contexts;
+
+        /// <inheritdoc />
+        public Contexts Contexts
+        {
+            get => _contexts ??= new Contexts();
+            set => _contexts = value;
+        }
+
+        [DataMember(Name = "user", EmitDefaultValue = false)]
+        private User? _user;
+
+        /// <inheritdoc />
+        public User User
+        {
+            get => _user ??= new User();
+            set => _user = value;
+        }
+
+        /// <inheritdoc />
+        [DataMember(Name = "environment", EmitDefaultValue = false)]
+        public string? Environment { get; set; }
+
+        /// <inheritdoc />
+        [DataMember(Name = "sdk", EmitDefaultValue = false)]
+        public SdkVersion? Sdk { get; internal set; } = new SdkVersion();
+
+        /// <inheritdoc />
+        [DataMember(Name = "fingerprint", EmitDefaultValue = false)]
+        [DontSerializeEmpty]
+        public IEnumerable<string> Fingerprint { get; set; } = Enumerable.Empty<string>();
+
+        /// <inheritdoc />
+        [DataMember(Name = "breadcrumbs", EmitDefaultValue = false)]
+        [DontSerializeEmpty]
+        public ICollection<Breadcrumb> Breadcrumbs { get; } = new SynchronizedList<Breadcrumb>();
+
+        /// <inheritdoc />
+        [DataMember(Name = "extra", EmitDefaultValue = false)]
+        [DontSerializeEmpty]
+        public IDictionary<string, object?> Extra { get; } = new ConcurrentDictionary<string, object?>();
```

Comment:
In the old implementation these lists were not allocated at all. Now we get these two lists instatiated for each new Scope pushed/poped even if nothing is ever added